### DISCUSSION TITLE
Add a custom button on Collection Interface page

### DISCUSSION THREAD

**Discussion by n-pn** [directus/directus#4216] [Archive]
For example I have a table name tickets that hold some ticket code to some events. I want to include a button that after click it it will send an email with the ticket code to the ticket holder.

The endpoint api for the sending event is done, the only problem is how to include it in the admin app.

**Comment by danieljwestman** [comment]
This one should do exactly (?) what you’re after:

Actions presentation interface
[https://github.com/directus/directus/pull/3677](https://github.com/directus/directus/pull/3677)

**Reply by n-pn** [reply]
This is exactly what I need!
Very appreciate it. Thank you very much.
